xds/internal/balancer/clusterresolver/configbuilder.go

```diff
@@ -190,7 +190,19 @@ func buildClusterImplConfigForEDS(g *nameGenerator, edsResp xdsresource.Endpoint
 		})
 	}
 
-	priorities := groupLocalitiesByPriority(edsResp.Localities)
+	var priorities [][]xdsresource.Locality
+	// Triggered by an EDS error before update, or empty localities list in a
```

Comment:
Or resource-not-found error?

Reply:
To me, EDS error meant resource not found || NACK. I'll change language to be explicit though.